logux-vue-todomvc/src/components/TodoItem.vue

```diff
@@ -0,0 +1,68 @@
+<template>
+  <li class="todo" :class="{ completed: todo.done, editing: editing }">
+    <div class="view">
+      <input class="toggle"
+        type="checkbox"
+        :checked="todo.done"
+        @change="toggleTodo(todo)">
+      <label v-text="todo.text" @dblclick="editing = true"></label>
+      <button class="destroy" @click="removeTodo(todo)"></button>
+    </div>
+    <input class="edit"
+      v-show="editing"
+      v-focus="editing"
+      :value="todo.text"
+      @keyup.enter="doneEdit"
+      @keyup.esc="cancelEdit"
+      @blur="doneEdit">
+  </li>
+</template>
+
+<script>
+export default {
+  name: 'Todo',
+  props: ['todo'],
```

Comment:
Since it's a real example, let's use [Vue’s Style Guide](https://vuejs.org/v2/style-guide/#Prop-definitions-essential).

> Prop definitions should be as detailed as possible.

It would make code clearer.